test/unit/org/apache/cassandra/index/sai/cql/CollectionIndexingTest.java

```diff
@@ -220,4 +222,9 @@ private void assertUnsupportedIndexOperator(int expectedSize, String query, Obje
         assertInvalidMessage(StatementRestrictions.REQUIRES_ALLOW_FILTERING_MESSAGE, query, values);
         assertEquals(expectedSize, execute(query + " ALLOW FILTERING").size());
     }
+
+    private static String createIndexDDL(String target)
```

Comment:
I think rename createIndexDDL to createSAIIndexDDL is more expressive

Reply:
It's an SAI test, but perhaps.

Reply:
1. `CREATE CUSTOM INDEX ON %s(VALUES(value)) USING 'StorageAttachedIndex'" `
2. `createIndexDDL(String target)` 


1 is more intuitive than 2, as I know I am creating SAI when I see StorageAttachedIndex, but I have no idea about what kind of index I am going to create when see "createIndexDDL".

Of course, you don’t need to change it if you do not want to, because this is just a  very small point, which is insignificant

Reply:
I think I prefer `createIdexDDL` because we are inside the SAI package. We have lots of mentions to "index" in that package and IMO it won't be practical to prefix all of them with "SAI" given the context.